### PR TITLE
Homework Node.js week2

### PR DESCRIPTION
**Pull request by OlenaKasian**

### CODE REVIEW

File: nodejs/week2/homework/warmup/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+});
+
+app.get("/numbers/multiply/:first/:second", (req, res) => {
+    const first = Number(req.params['first']);
+    const second = Number(req.params['second']);
+    const multiply = first*second;
+    res.send(multiply.toString())
+})
+
+
```
Comment (laila1222) [inline nodejs/week2/homework/warmup/index.js:22]: Good job!

File: nodejs/week2/homework/meal-sharing/src/backend/api/meals-router.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    const requestedMeal = meals.find(meal => meal.id === Number(request.params['id']));
+    if (requestedMeal) {
+      response.send(requestedMeal)
+    }
+    response.send(`There is no meal with '${request.params['id']}' id`)
+  } catch (error) {
+    throw error;
+  }
+});
+
```
Comment (laila1222) [inline nodejs/week2/homework/meal-sharing/src/backend/api/meals-router.js:48]: Well done!

File: nodejs/week2/homework/meal-sharing/src/backend/api/reservations-router.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+router.get("/:id", async (request, response) => {
+  try {
+    const requestedReservation = reservations.find(reservation => reservation.id === Number(request.params['id']));
+    response.send(requestedReservation)
+  } catch (error) {
+    throw error;
+  }
+});
+
```
Comment (laila1222) [inline nodejs/week2/homework/meal-sharing/src/backend/api/reservations-router.js:22]: Good!

File: nodejs/week2/homework/meal-sharing/src/backend/api/reviews-router.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+router.get("/:id", async (request, response) => {
+  try {
+    const requestedReview = reviews.find(review => review.id === Number(request.params['id']));
+    response.send(requestedReview)
+  } catch (error) {
+    throw error;
+  }
+});
+
```
Comment (laila1222) [inline nodejs/week2/homework/meal-sharing/src/backend/api/reviews-router.js:22]: Well done

**Review by laila1222 — APPROVED** [review]
Excellent homework. Congrats! All endpoints seem to work as they are supposed to. 
Keep up the good work! 👍 💯

File: nodejs/week2/homework/meal-sharing/src/backend/api/reviews-router.js (hunk 4 above)
Comment (OlenaKasian) [reply nodejs/week2/homework/meal-sharing/src/backend/api/reviews-router.js:22]: Thank you Laila for your code review!

**Pull request merged**